app/src/main/java/com/quran/labs/androidquran/presenter/data/QuranDataPresenter.kt

```diff
@@ -203,7 +203,7 @@ class QuranDataPresenter @Inject internal constructor(
       }
 
       val pageType = quranSettings.pageType
-      if (!quranSettings.didCheckPartialImages(pageType)) {
+      if (!quranSettings.didCheckPartialImages(pageType) && !pageType.endsWith("lines")) {
```

Comment:
moving forward, i don't want to support this partial pages concept nor downloading pages on the fly - so for new page types (lines, etc), just don't run the partial page checker. in the future, we should do this for all page types.